testsuite/features/secondary/proxy_container_cobbler_pxeboot.feature

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) 2021-2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# We also test 'Bootstrapping using the command line' in this feature with the following script:
+# https://github.com/uyuni-project/uyuni/blob/master/java/conf/cobbler/snippets/minion_script
+
+@skip_if_github_validation
+@containerized_server
+@proxy
+@private_net
+@pxeboot_minion
+@scope_cobbler
+Feature: PXE boot a terminal with Cobbler and containerized proxy
+  In order to automate client system installations with a containerized proxy
+  As the system administrator
+  I want to PXE boot one host with Cobbler
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+    And I am on the Systems overview page of this "proxy"
+
+  Scenario: Start Cobbler monitoring
+    When I start local monitoring of Cobbler
+
+  # We currently test Cobbler with SLES 15 SP4, even on Uyuni
+  Scenario: Install the TFTP boot package on the server for Cobbler tests
+    When I install package tftpboot-installation on the server
+    And I wait for "tftpboot-installation-SLE-15-SP4-x86_64" to be installed on "server"
+
+  Scenario: Create auto installation distribution
+    When I follow the left menu "Systems > Autoinstallation > Distributions"
+    And I follow "Create Distribution"
+    And I enter "SLE-15-SP4-TFTP" as "label"
+    And I enter "/usr/share/tftpboot-installation/SLE-15-SP4-x86_64/" as "basepath"
+    And I select "SLE-Product-SLES15-SP4-Pool for x86_64" from "channelid"
+    And I select "SUSE Linux Enterprise 15" from "installtype"
+    And I click on "Create Autoinstallable Distribution"
+    Then I should see a "Autoinstallable Distributions" text
+    And I should see a "SLE-15-SP4-TFTP" link
+
+  # WORKAROUND bsc#1195842
+  # Default cobbler kernel parameters are wrong in case of proxy
+  Scenario: Fix kernel parameters
+    When I follow the left menu "Systems > Autoinstallation > Distributions"
+    And I follow "SLE-15-SP4-TFTP"
+    And I enter "useonlinerepo insecure=1 install=http://proxy.example.org/ks/dist/SLE-15-SP4-TFTP self_update=http://proxy.example.org/ks/dist/child/sle15-sp4-installer-updates-x86_64/SLE-15-SP4-TFTP" as "kernelopts"
```

Comment:
Quite ugly step :/ 

But it's hard to find a better solution.. as it all needs to go into one line, and triple quotes will not help that much:

```
And I set the text for "kernelopts":
  """
  useonlinerepo insecure=1 install=http://proxy.example.org/ks/dist/SLE-15-SP4-TFTP self_update=http://proxy.example.org/ks/dist/child/sle15-sp4-installer-updates-x86_64/SLE-15-SP4-TFTP
  """
```

Reply:
Indeed. If I get 2 seconds, I create a dedicated step hiding all those gory technical details.

Reply:
> Btw. 4.3 (if applicable here) is already fixed. Uyuni/5.0 waiting until branch is unlocked.

Are you referring to https://bugzilla.suse.com/show_bug.cgi?id=1221889 ?

If yes, bsc#1221889 was happening *despite* this correction.

Also, this correction was done before bsc#1221889: it was not a workaround for it, but for some more ancient issue.

However, @aaannz , if you think this line should change nevertheless, can you please tell me what your suggested correction would be? Thanks a lot in advance.

Reply:
Ended up using triple quotes. But a dedicated step might having been better.